subhub/customer.py

```diff
@@ -101,7 +106,13 @@ def subscribe_customer(customer: stripe.Customer, plan_id: str) -> stripe.Subscr
     :param plan:
     :return: Subscription Object
     """
-    return stripe.Subscription.create(customer=customer, items=[{"plan": plan_id}])
+    sub = None
+    try:
+        sub = stripe.Subscription.create(customer=customer, items=[{"plan": plan_id}])
```

Comment:
super ugly function signature that requires you to pass stuff like this: `items=[{"plan": plan_id}]`.  Is this necessary? Beneficial?